big_tests/tests/graphql_helper.erl

```diff
@@ -84,10 +90,15 @@ init_user(Config) ->
 
 init_domain_admin_handler(Config) ->
     Domain = domain_helper:domain(),
-    Password = base16:encode(crypto:strong_rand_bytes(8)),
-    Creds = {<<"admin@", Domain/binary>>, Password},
-    ok = domain_helper:set_domain_password(mim(), Domain, Password),
-    add_specs([{domain_admin, Creds}, {schema_endpoint, domain_admin} | Config]).
+    case (not ct_helper:is_ct_running())
```

Comment:
I think that in big tests we always expect CT to be running. Anyway, this condition does not make sense to me, as it does not mean that we don't have to check the DB.